sparkle/src/components/Button.tsx

```diff
@@ -186,3 +186,16 @@ export function Button({
     </Tooltip>
   );
 }
+
+interface ButtonBarProps {
+  children: ReactNode;
+  className?: string;
+}
+
+Button.List = function ({ children, className }: ButtonBarProps) {
```

Comment:
I'm not sure I understand what this is -- is it essentially just a list of buttons ?

Reply:
Exactly